sdk/monitor/Azure.Monitor.Query/src/MetricsClient.cs

```diff
@@ -24,11 +24,11 @@ public class MetricsClient
         /// Creates an instance of <see cref="MetricsClient"/>.
         /// <code snippet="Snippet:CreateMetricsClient" language="csharp">
         /// var client = new MetricsClient(
-        ///     new Uri(&quot;https://metrics.monitor.azure.com/.default&quot;),
+        ///     new Uri(&quot;https://&lt;region&gt;.metrics.monitor.azure.com/&quot;),
         ///     new DefaultAzureCredential());
         /// </code>
         /// </summary>
-        /// <param name="endpoint">The data plane service endpoint to use. For example, <c>https://metrics.monitor.azure.com/.default</c> for Azure Public Cloud.</param>
+        /// <param name="endpoint">The data plane service endpoint to use. For example, <c>https://(region).metrics.monitor.azure.com/</c> for Azure Public Cloud.</param>
```

Comment:
For consistency in placeholder syntax, use less than/greater than symbols
```suggestion
        /// <param name="endpoint">The data plane service endpoint to use. For example, <c>https://&lt;region&gt;.metrics.monitor.azure.com/</c> for Azure Public Cloud.</param>
```